composer/spec/dependabot/composer/update_checker/version_resolver_spec.rb

```diff
@@ -201,6 +201,15 @@
       it { is_expected.to eq(Dependabot::Composer::Version.new("1.25.1")) }
     end
 
+    context "with a project that specifies a platform package" do
```

Comment:
I am not sure about this test. When I hardcode the return value in the Helpers.composer_version either v1 or v2 the test passes. Shouldn't it verify that the correct composer version is used?

Reply:
You're right, thanks! I've added two different tests to make sure the new behavior is covered